editor/import/resource_importer_scene.cpp

```diff
@@ -1202,10 +1202,15 @@ Node *ResourceImporterScene::_post_fix_node(Node *p_node, Node *p_root, HashMap<
 				r_scanned_meshes.insert(m);
 			}
 
+			if ((!node_settings.has("generate/physics") || (node_settings.has("generate/physics") && int(node_settings["generate/physics"]) == PHYSICS_DEFAULT)) && p_default_collisions == CollisionMode::COLLISION_STATIC) {
+				node_settings["generate/physics"] = PHYSICS_ENABLE;
```

Comment:
My mistake here, node_settings should be const, since this Dictionary is passed by the import dialog which uses to edit it.

Please find a way to do this without overriding the node settings.

Reply:
I looked over this code and unless I'm incorrect, the node_settings actually *aren't* the ones which get passed to the import dialog. On line 1112, this gets called:
```
//make sure this is unique
node_settings = node_settings.duplicate(true);
```
Could you double check that this criticism is valid because I think PR would be very worth having for the release candidate as it can significantly streamline import workflow for 3D